scripts/darwin.mk

```diff
@@ -1,11 +1,16 @@
 # Build Tools for OSX
-CC := gcc-10 -fdiagnostics-color -fmax-errors=5
-CPP := g++-10 -fdiagnostics-color -fmax-errors=5
+GCC_VERSION = 10
+ifeq ($(shell uname -p),arm)
+  GCC_VERSION = 11
+endif
+
+CC := gcc-$(GCC_VERSION) -fdiagnostics-color -fmax-errors=5
```

Comment:
It was hard to come up with anything better here. For some reason homebrew gcc@11 on i386 is unable to find certain include files... I couldn't figure out why.

This works for now, we have to use gcc@11 on arm since there is no gcc@10 build.

Reply:
Added this info as a comment for posterity.